src/components/Card/index.js

```diff
@@ -15,34 +10,34 @@ const Card = ({ href, children, image, ...props }) => {
   const classes = useStyles(props);
 
   return (
-    <MuiCard className={classes.root}>
```

Comment:
What's happening in this component?

Reply:
@kilemensi For this section, noticed that using cards drowns out box shadow applied on it for the insight class cards.So I used `div` instead to fix that

Reply:
`CardActionArea` is there to support clicking/linking to actual story so we should not remove it. You can disable button ripple effect [global](https://material-ui.com/getting-started/faq/#how-can-i-disable-the-ripple-effect-globally) and you should also be able to style `&:hover` on the component if need be.